R/utils.R

```diff
@@ -69,11 +69,18 @@ MakeMS2CosineDataframe <- function(df) {
 #' Intensity is scaled to 100 and filtered to drop all intensity values below 0.5.
 #'
 MakeScantable <- function(scan) {
+  # Variable names here could also be improved - this converts an MS2 string to an MS2 data frame, right? "Scan" is not super informative
   requireNamespace("dplyr", quietly = TRUE)
+  # Not sure I love this method of converting string to df - something like
```

Comment:
I agree that the read table isn't ideal. It's driving me nuts but I can't understand why the code doesn't work in the package with your suggested changes. I have left it for now but I will revisit it to make it more elegant later. For now I think it's a small thing that can be left.